src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -244,5 +245,25 @@ public StudyController(StudyService studyService) {
             return ResponseEntity.ok().headers(header).contentType(MediaType.APPLICATION_OCTET_STREAM).body(exportNetworkInfos.getNetworkData());
         });
     }
+
+    @PostMapping(value = "/{userId}/studies/{studyName}/loadflow/parameters/set")
```

Comment:
This is not a standard REST API design just a post on /{userId}/studies/{studyName}/loadflow/parameters